addon/components/admin-input.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
 
-var set = Ember.set;
-var get = Ember.get;
-var computed = Ember.computed;
+const set = Ember.set;
+const get = Ember.get;
+const computed = Ember.computed;
```

Comment:
Even without destructuring ```ember test``` is not running. The output is this:
```
version: 1.13.1
Could not find watchman, falling back to NodeWatcher for file system events.
Visit http://www.ember-cli.com/#watchman for more info.
Building.DEPRECATION: Using `{{view}}` or any path based on it (L4:C2) has been deprecated. [deprecation id: view.keyword.view] See http://emberjs.com/deprecations/v1.x#toc_view-and-controller-template-keywords for more details.
        at deprecatePath (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:11280:32)
        at /home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:11248:7
        at Walker.visit (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:17985:7)
        at visitors.Program (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:17995:16)
        at Walker.children (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:18018:7)
        at Walker.visit (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:17986:12)
        at visitors.BlockStatement (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:18007:14)
        at Walker.children (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:18018:7)
        at Walker.visit (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:17986:12)
DEPRECATION: Using `{{view}}` or any path based on it (L4:C2) has been deprecated. [deprecation id: view.keyword.view] See http://emberjs.com/deprecations/v1.x#toc_view-and-controller-template-keywords for more details.
        at deprecatePath (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:11280:32)
        at /home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:11248:7
        at Walker.visit (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:17985:7)
        at visitors.Program (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:17995:16)
        at Walker.children (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:18018:7)
        at Walker.visit (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:17986:12)
        at visitors.BlockStatement (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:18007:14)
        at Walker.children (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:18018:7)
        at Walker.visit (/home/clairton/Projetos/ember-admin_/bower_components/ember/ember-template-compiler.js:17986:12)
Built project successfully. Stored in "/home/clairton/Projetos/ember-admin_/tmp/class-tests_dist-Ww0e1eK9.tmp".
```
And the cursor stay there, :(